src/lib/autocomplete/autocomplete.ts

```diff
@@ -35,7 +34,7 @@ export type AutocompletePositionY = 'above' | 'below';
 export class MdAutocomplete implements AfterContentInit {
 
   /** Manages active item in option list based on key events. */
-  _keyManager: ActiveDescendantKeyManager;
+  _keyManager: _ActiveDescendantKeyManager;
```

Comment:
We can totally remove them, but the idea (from angular core) was to not "expose" those things to the public.

Though we don't have much stuff in our `private_exports` file, so we would be fine by just exposing those?